server/src/main/java/org/opensearch/index/SegmentReplicationShardStats.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.common.unit.ByteSizeValue;
+import org.opensearch.common.unit.TimeValue;
+import org.opensearch.core.xcontent.ToXContentFragment;
+import org.opensearch.core.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+/**
+ * SegRep stats for a single shard.
+ *
+ * @opensearch.internal
+ */
+public class SegmentReplicationShardStats implements Writeable, ToXContentFragment {
+    private final String allocationId;
+    private final long checkpointsBehindCount;
+    private final long bytesBehindCount;
+    private final long currentReplicationLag;
+    private final long lastCompletedLag;
```

Comment:
What does it mean ?

Reply:
Updated to better names - 
`currentReplicationTimeMillis` and `lastCompletedReplicationTimeMillis`